deploy/kubernetes/webhook-example/webhook.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       serviceAccountName: snapshot-webhook
       containers:
       - name: snapshot-validation
-        image: k8s.gcr.io/sig-storage/snapshot-validation-webhook:v5.0.1 # change the image if you wish to use your own custom validation server image
+        image: k8s.gcr.io/sig-storage/snapshot-validation-webhook:v6.0.1 # change the image if you wish to use your own custom validation server image
```

Comment:
Can you change this to registry.k8s.io/sig-storage/snapshot-validation-webhook:v6.0.1?

Reply:
Done